bundles/bundle.sh

```diff
@@ -6,7 +6,7 @@ TARGET="./"
 (cd $ROOT && mvn clean -P all)
 (cd $ROOT && mvn package -P all -Dmaven.test.skip=true)
 
-rm -rf $TARGET
+rm -rf "./"
```

Comment:
I think there is no good justification for this. It might remove itself, force removal should not be needed and the use case is so no old (now renamed) bundles are deleted. Since a rename requires you to alter this script I do not see the issue with manually removing a file from the filesystem.